Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: sync
 sync:
-	python sync/sync.py
+	python sync/sync.py || python3 sync/sync.py
```

Comment:
:sweat_smile: can we just use `python` and fail in `python --version` is not 3.x ? (in a pre-target)